src/modules/dashboard/components/ExpenditurePage/Split/Split.css

```diff
@@ -0,0 +1,38 @@
+@value paddingRightLeft: 54px;
+
+.splitContainer {
+  padding: 0 paddingRightLeft 80px;
+}
+
+.split {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 6px;
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  line-height: 18px;
+  color: color-mod(var(--dark), alpha(85%));
```

Comment:
There shouldn't be any comma before alpha. There is a warning in console because of it now that will disappear if you remove the comma.
<img width="768" alt="Screenshot 2022-07-20 at 23 04 32" src="https://user-images.githubusercontent.com/34057551/180071675-e6796f6c-1d67-405e-aec9-80a8ed47e5ae.png">